### ISSUE TITLE
Sim translation updates for common string changes

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/rosetta#197]
@jbphet, I'm curious if a common string change submitted for some sim that includes common string changes (say, a JOIST string that affects all sims) would trigger a build/update for ALL sims that include that string?

A brief scan of the code (and my memory) would think that only the sim that was "submitted" would get rebuilt with the translation.

During dev meeting today, it was discussed that we might WANT to rebuild all sims/locales that are affected by the string changes, not just the submitted sim.

**Comment by jbphet** [comment]
Yes @jonathanolson, your brief scan and memory are correct, a change to common code does not trigger a rebuild of all sims that use it.  We would have to do a maintenance release of the sims that use that string to force the English and translated versions to be rebuilt.

**Comment by jonathanolson** [comment]
Presumably we'll discuss at the next dev meeting about whether this should change. Unassigning.

**Comment by jbphet** [comment]
This is doable, but not trivial, because the way we figure out which common code strings are actually used by a sim is to download the published HTML file and analyze, so we'd need to do that for every published sim with every change to a common code string.

We discussed at a dev meeting today and decided to defer until after the support for translation of the app is complete.

**Comment by oliver-phet** [comment]
Adding a comment so that I get notifications for this issue.

**Comment by jbphet** [comment]
For future reference, there was an older issue where this was discussed when we were first creating Rosetta, see #43.
